apps/constants/base.py

```diff
@@ -454,6 +454,7 @@
 REVIEWED_LP_FULL = 30
 REVIEWED_LP_PRELIM = 31
 REVIEWED_LP_UPDATE = 32
+REVIEWED_OVERDUE_BONUS = 2
```

Comment:
Maybe also add this to the `REVIEWED_CHOICES` dict? The same way you added it to the `REVIEWED_SCORE` one?